lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -425,6 +429,91 @@ sub _eu_xml {
     };
 }
 
+sub _unsc_xml {
+    my ($xml_content) = @_;
+
+    # Preprocess the XML content to escape unescaped ampersands
+    $xml_content =~ s/&(?!(?:amp|lt|gt|quot|apos);)/&amp;/g;
+    my $data = xml2hash($xml_content, array => ['INDIVIDUAL', 'INDIVIDUAL_ALIAS', 'INDIVIDUAL_ADDRESS', 'INDIVIDUAL_DATE_OF_BIRTH', 'INDIVIDUAL_PLACE_OF_BIRTH', 'INDIVIDUAL_DOCUMENT'])->{CONSOLIDATED_LIST};
+
+    # Extract the dateGenerated attribute from the first line of the XML content
+    my ($date_generated) = $data->{'-dateGenerated'};
+    die "Corrupt data. Release date is missing\n" unless $date_generated;
+
+    # Convert the dateGenerated to epoch milliseconds
+    my $publish_epoch = _date_to_epoch($date_generated // '');
+
+    my $dataset = [];
+
+    for my $individual (@{$data->{'INDIVIDUALS'}->{'INDIVIDUAL'}}) {
+        my %entry;
+
+        $entry{first_name}           = $individual->{'FIRST_NAME'};
+        $entry{second_name}          = $individual->{'SECOND_NAME'};
+        $entry{third_name}           = $individual->{'THIRD_NAME'}           // '';
+        $entry{fourth_name}          = $individual->{'FOURTH_NAME'}          // '';
+        $entry{name_original_script} = $individual->{'NAME_ORIGINAL_SCRIPT'} // '';
+
+        my @names = (
+            $entry{first_name}           // '',
+            $entry{second_name}          // '',
+            $entry{third_name}           // '',
+            $entry{fourth_name}          // '',
+            $entry{name_original_script} // '',
+        );
+
+        foreach my $alias (@{$individual->{INDIVIDUAL_ALIAS}}) {
+            if (ref($alias->{'ALIAS_NAME'}) ne 'HASH' || %{$alias->{'ALIAS_NAME'}}) {
+                push @names, $alias->{'ALIAS_NAME'} // '';
+            }
+        }
+
+        my @dob_list = ($individual->{'INDIVIDUAL_DATE_OF_BIRTH'}[0]{'DATE'} // '');
```

Comment:
In some cases the exact date isn't provided. So, `{'INDIVIDUAL_DATE_OF_BIRTH'}[0]{'DATE'}` may not be available. 
In all of the following cases, the date of birth will be an empty string.

`<INDIVIDUAL_DATE_OF_BIRTH>
<TYPE_OF_DATE>EXACT</TYPE_OF_DATE>
<YEAR>1958</YEAR>
</INDIVIDUAL_DATE_OF_BIRTH>`

`<INDIVIDUAL_DATE_OF_BIRTH>
<TYPE_OF_DATE>APPROXIMATELY</TYPE_OF_DATE>
<YEAR>1958</YEAR>
</INDIVIDUAL_DATE_OF_BIRTH>`


`<INDIVIDUAL_DATE_OF_BIRTH>
<TYPE_OF_DATE>BETWEEN</TYPE_OF_DATE>
<FROM_YEAR>1972</FROM_YEAR>
<TO_YEAR>1973</TO_YEAR>
</INDIVIDUAL_DATE_OF_BIRTH>`

Reply:
done